pkg/bpf/config.go

```diff
@@ -40,31 +40,23 @@ func init() {
 }
 
 type Config struct {
-	Mode                string
-	BpfFsPath           string
-	Cgroup2Path         string
-	EnableKmesh         bool
-	EnableKmeshWorkload bool
-	EnableMda           bool
-	BpfVerifyLogSize    int
+	Mode             string
+	BpfFsPath        string
+	Cgroup2Path      string
+	EnableMda        bool
+	BpfVerifyLogSize int
 }
 
 func (c *Config) AttachFlags(cmd *cobra.Command) {
 	cmd.PersistentFlags().StringVar(&c.BpfFsPath, "bpf-fs-path", "/sys/fs/bpf", "bpf fs path")
 	cmd.PersistentFlags().StringVar(&c.Cgroup2Path, "cgroup2-path", "/mnt/kmesh_cgroup2", "cgroup2 path")
-	cmd.PersistentFlags().StringVar(&c.Mode, "mode", "ads", "controller plane mode, valid values are [ads, workload]")
+	cmd.PersistentFlags().StringVar(&c.Mode, "mode", "workload", "controller plane mode, valid values are [ads, workload]")
```

Comment:
To prevent misconfiguration, it would be better to check if the `mode` value is expected. 

Because other code depends on this value, it can only be `ads` or `workload`.

Reply:
we can add a validation in the parse funcction